ios/PurchasesHybridCommon/PurchasesHybridCommon/RCErrorContainer.m

```diff
@@ -28,8 +28,14 @@ - (instancetype)initWithError:(NSError *)error extraPayload:(NSDictionary *)extr
     }
 
     if (error.userInfo[RCReadableErrorCodeKey]) {
-        dict[@"readableErrorCode"] = error.userInfo[RCReadableErrorCodeKey];
-        dict[@"readable_error_code"] = error.userInfo[RCReadableErrorCodeKey];
+        NSString *readableErrorCode = error.userInfo[RCReadableErrorCodeKey];
+        dict[@"readableErrorCode"] = readableErrorCode;
+        dict[@"readable_error_code"] = readableErrorCode;
+            
+        NSMutableDictionary *fixedUserInfo = [NSMutableDictionary dictionaryWithDictionary:error.userInfo];
+        fixedUserInfo[@"readableErrorCode"] = readableErrorCode;
```

Comment:
should we add a comment explaining _why_ this is a fix for context for future devs?

Reply:
Sure! I also wanted to clarify that I solved it this way because the other fix would be to change the `RCReadableErrorCodeKey` and that would require updating all hybrids. This way we can just fix it for now, and think about a better solution for errors in that project we have to fix error handling in the hybrids 😄